content/en/blog/_posts/2024-03-19-go-workspaces.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: blog
+title: 'Using Go workspaces in Kubernetes'
+date: 2024-03-19T08:30:00-08:00
+slug: go-workspaces-in-kubernetes
+canonicalUrl: https://www.kubernetes.dev/blog/2024/03/19/go-workspaces-in-kubernetes/
```

Comment:
```suggestion
```
If we're doing this as a post-release blog, I wouldn't link it to the contributor site article from March (at least, not using `<link rel="canonical" … >`)

Reply:
Accepted